docs/user/filesystem.md

```diff
@@ -94,15 +100,16 @@ accessed in balanced mode only. If the container was created with relaxed mode,
 it can be accessed in relaxed or balanced mode. In either mode, there is a
 consistency semantic issue that is not properly handled:
 
-* Open-unlink semantics: This occurs when a client obtains an open handle on an
-  object (file or directory), and accesses that object (reads/writes data or
-  create other files), while another client removes that object that the other
-  client has opened from under it. In DAOS, we don't track object open handles
-  as that would be very expensive, and so in such conflicting cases, the worst
-  case scenario is the lost/leaked space that is written to those orphan objects
-  that have been unlinked from the namespace.
+* Open-unlink semantics: This occurs when a client obtains an open handle on an object (file or
+  directory), and accesses that object (reads/writes data or create other files), while another
+  client removes that object that the other client has opened from under it. In DAOS, we don't track
+  object open handles as that would be very expensive, and so in such conflicting cases, the worst
+  case scenario is the lost/leaked space that is written to those orphan objects that have been
+  unlinked from the namespace. DAOS implements a file system checker that can be used to either
+  relink those orphaned objects back in a lost+found directory or remove them from the container to
+  reclaim the space.
```

Comment:
I would add this text to the list of POSIX limitations above since it is not really relevant to the balanced vs relaxed mode discussion.

Reply:
good point

Reply:
fixed